thewalrus/.ipynb_checkpoints/_hafnian-checkpoint.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
```

Comment:
@ziofil don’t forget to delete the `.vscode`, `.ipynb_checkpoints`, and `.DS_STORE` directories before merging!